.github/workflows/main.yml

```diff
@@ -93,7 +93,7 @@ jobs:
         run: |
           cargo llvm-cov clean --workspace
           cargo llvm-cov --no-report
-          cargo llvm-cov --no-run --lcov --ignore-filename-regex "libseccomp/src|integration_test/src|test_framework/src|systemd_api.rs" --output-path ./coverage.lcov
+          cargo llvm-cov --no-run --lcov --ignore-filename-regex "libseccomp/src|crates/integration_test/src|test_framework/src|systemd_api.rs" --output-path ./coverage.lcov
```

Comment:
If this change was done only to fix the CI coverage issue, this can be reverted, as that seems nightly issue, not the PR changes, and previous (without this change) works fine without this change.

Reply:
This change doesn't seems to fix it 😂

Reply:
As I said in https://github.com/containers/youki/pull/614#issuecomment-1013831105 I think the issue is not in your changes issue is in the current nightly rust version, with which CI is running this.

If you haven't updated your rust using `rustup update` in at least last 5-6 days, try running cargo-llvm-cov manually using the commands we use in CI. I did that for a nightly version of roughly 1 month before. The coverage report was generated normally. Then I updated the rust , and ran it again, and with the updated nightly version I got the exact same error we are getting in this PR.

As it seems now, only way is to wait next few days, in which the updated nightly might fix this issue. I'm thinking of opening an issue regarding this, but not sure if it should be filed on cargo-llvm-cov or on rust itself.

Reply:
@YJDoc2 Thanks for your information. I have put them back.